Resources

Nightfire-0x07 opened the pull request:
Adding resource's page to a separate pull request from a dedicated branch.

daylamtayari commented:
The `GetConnected` file looks good and that has already being approved in #8, just waiting on the merge.

daylamtayari reviewed (changes requested):
On the right track but please integrate the changes requested.
Also add a description to each site linked and again remove the link blue colour.
Concerning the description, just a short 1 sentence long description of each thing because as someone new into cybersec, looking at this and seeing 'Port Swigger' for example, I have no clue what the hell that is. 
Also if you could add a guides section with guides on the basic stuff such as setting up an environment (installing virtual box, setting up kali/parrot, etc.) that would be appreciated.
To complement it (as so to have symmetric columns) I think a tools column would be nice with a list of recommended tools or websites such as nmap, or revshells.com, etc. you get the gist. If you have a better idea though please fell free to propose it.

KyleMuncie reviewed (dismissed):
I agree with Tayari's reviews. Other than that, it looks good to me! Nice work

Nightfire-0x07 commented:
RE: commit 8f924ea91bb0d4432607424edf9bfb9ff5218c1d 

Forgot to add that I also fixed the highlighted section to be smaller and I removed the blue link color.

daylamtayari commented on Resources.html:
```diff
@@ -81,25 +101,65 @@
+            <a href="https://portswigger.net/" target="_blank"><b>Port Swigger</b></a><p>A free web security based platform that includes a web academy to learn almost every concept regarding website hacking and practice in vulnerable labs. Burp Suite is the main tool used for these labs, and there is a free community edition for learning purposes.</p>
+            <a href="https://tryhackme.com/" target="_blank"><b>Try Hack Me</b></a><p>Free training platform that has individual rooms to learn each security concept as it relates to web, cryptography, Windows, Linux, and software like Wireshark, Hydra, and Nessus. Similar to Hack the Box, but it is considered easier and better for beginners.</p>
+            <a href="https://cyberdefenders.org/" target="_blank"><b>Cyber Defenders</b></a><p>A training platform with a focus on defending networks from malicious attacks.</p> 
+            <a href="https://blueteamlabs.online/" target="_blank"><b>Blue Team Labs</b></a><p>A similar platform to train in defensive techniques related to security investigations.</p>
```
Just a general comment but don't feel obligated to avoid repetition, if you want to say "A training platform in defensive..." go ahead.

daylamtayari commented on Resources.html:
```diff
@@ -81,25 +101,65 @@
+            <h3>Where to learn Linux</h3>
+            </div>
+            <a href="https://linuxjourney.com/" target="_blank"><b>Linux Journey</b></a><p>An easy-to-follow guide on everything you should know about Linux that comes with helpful examples to reinforce your knowledge.</p>
+            <a href="https://www.linuxcommand.org/" target="_blank"><b>Linux Command</b></a><p>Focuses mainly on terminal commands, shell scripts, and learning to get a handle on optimizing your experience using Linux.</p>
```
Good description but what is that, if you could just specify that at the beginning that would be good (i.e. a guide, a wiki, etc.).

daylamtayari commented on Resources.html:
If you could change the first "and" to a comma, would make more grammatical sense.

daylamtayari commented on Resources.html:
Slightly confused about the following bit:
> along with myriad resources of learning material...
Just the wording feels weird and slightly confusing.
Did you mean to say "along with a myriad of resources, learning material..."?

daylamtayari commented on Resources.html:
Overall great descriptions!

daylamtayari commented on Resources.html:
```diff
@@ -81,25 +101,65 @@
+            <div class="div-highlight">
+            <h3>Where to practice</h3>
+            </div>
+            <a href="https://overthewire.org/wargames/bandit/" target="_blank"><b>Over the Wire</b></a><p>A free series of wargames starting with "Bandit" that help you learn and practice security concepts in successive stages that build on each other.</p>
```
A comment which applies to all of these descriptions/sections, change the `href` from being the title/name of the tool to the entire div, as such when a user clicks on that div for that particular resource, that page is opened for them.

Nightfire-0x07 commented:
Just making one comment to address the above comments with changes made:
1. Padding - margin: instead of adding an extra column, I just added padding to the row using a class. Let me know if this doesn't work for you.
2. Text - left is now text-center for rows
3. Bandit phrase removed
4. Fixed grammatical error with the extra "and" and kept "training platform" for the blue team labs description
5. Removed "myriad resources" and replaced with simpler description
6. Made href apply to the <p> tag, since otherwise I would have needed to add <div> elements to each type/description. I also think it makes it more clear that there is a link embedded using the p:hover css styling. Let me know if this doesn't work for you and I'll go back to adding the <div> for the entire title/description
7. added target="_blank" to <head>

daylamtayari commented on Resources.html:
```diff
@@ -81,25 +111,75 @@
+                    <h3>Where to practice</h3>
+                    </div>
+              </div>
+            <a href="https://overthewire.org/wargames/bandit/">
```
Implement all of the resources into a div within the `a` tag to allow for the entire div to be clickable and not just the row or title. Also it would allow us to stylize the page more easily when we get to that stage.
For example, this one would become:
 ```suggestion
             <a href="https://overthewire.org/wargames/bandit/">
                <div>
                    <b>Over the Wire</b>
                    <p>A free series of wargames that help you learn and practice security concepts in successive stages that build on each other.</p>
                </div>
             </a>
```

daylamtayari commented on Resources.html:
```diff
@@ -55,7 +57,35 @@
+    }
+
+    .row{
+        padding: 10px;
```
Increase the padding to a value around `50px` as discussed.

daylamtayari reviewed (changes requested):
Overall very good, just got some two very minor changes to do and otherwise, I think it would be good for approval.

daylamtayari commented on Resources.html:
```diff
@@ -55,7 +57,29 @@
+    }
+
+    .row{
+        padding: 50px;
```
We should want a lower `padding-top` and `padding-bottom` since that would look much better imo but that is part of the stylistic design of the website and not functional so will be addressed in the future at a later stage of development.

daylamtayari reviewed (approved):
Looks good! Great work on this page overall.
Only thing worth mentioning from the last changes was a minor stylistic issue however, it has no functional effects whatsoever so will be addressed at a later stage of the project's development.

KyleMuncie reviewed (approved):
Looks good. Were you not able to look @ the header and the footer code? Or was this before that was all done?
